doc/output-details-and-examples/src/output-files/eplusout-eio.tex

```diff
@@ -1921,22 +1921,7 @@ \subsubsection{Field: Nominal U (with film coefs)}\label{field-nominal-u-with-fi
 \bottomrule
 \end{longtable}
 
-NOTE:~ The resulting ``Nominal U with film coefs'' is only for comparison purposes with the ASHRAE 90.1 requirements. Actual film coefficients are calculated during the simulation and are dependent on several factors that may change during the simulation time period.
-
-For windows, the ``Nominal U with film coefs'' is calculated using the following assumptions from the WINDOW 5 program (also NFRC winter conditions):
-
-\begin{itemize}
-\item
-  Indoor air temperature = 21 C (69.8 F)
-\item
-  Outdoor air temperature = -18 C (-.4 F)
-\item
-  Exterior film coefficient = ~ 26 W/m\(^{2}\) (4.58 ft\(^{2}\)-F-hr/BTU)
-\item
-  Interior film coefficient = Calculated by EnergyPlus convection algorithm
-\item
-  No incident solar beam radiation
-\end{itemize}
+NOTE: The resulting ``Nominal U with film coefs'' is only for comparison purposes with the ASHRAE 90.1 requirements. Actual film coefficients are calculated during the simulation and are dependent on several factors that may change during the simulation time period.  It should be noted that the Nominal U-values with film coefficients that are listed in this report do not account for whether or not the surface in question is considered a "semi-exterior" surface. For such "semi-exterior" surfaces, use the Nominal U-value without film coefficients listed and then factor in the film coefficients for such surfaces listed in the literature (e.g., ASHRAE Standard 90.1).  For surfaces that use Other Side Coefficients, the exterior film coefficient that is used in this calculation is based on the value entered by the user in the Other Side Coefficients input.
```

Comment:
I'm not going to request it be changed now, but in the future, please avoid raw quotation marks in the LaTeX source.  Use two backticks to start the quote and two apostrophes to end it.